brave/src/test/java/brave/TracingTest.java

```diff
@@ -112,7 +112,7 @@
 
   @Test public void spanHandler_loggingByDefault() {
     try (Tracing tracing = Tracing.newBuilder().build()) {
-      assertThat(tracing.tracer().pendingSpans).extracting("spanHandler.delegate")
+      assertThat((Object) tracing.tracer().pendingSpans).extracting("spanHandler.delegate")
```

Comment:
Assertion changed from `ObjectAssert` to `ListAssert`. Other ways to fix

- Don't implement `Iterable` in `WeakConcurrentMap`
- Stop extending `WeakConcurrentMap` in `PendingSpans`

Could do now or in the future when integrating upstream's library in.